app/serializers/review_serializer.rb

```diff
@@ -0,0 +1,6 @@
+class ReviewSerializer < ActiveModel::Serializer
+  attributes :id, :rating, :body
+  
+  belongs_to :band
+  belongs_to :user
```

Comment:
you add 4 and 5 here, but you dont add them to line 2. further, you likely dont need line 4, since you probably dont want the band to be added in the json response (you likely already have it in state or somewhere else in the response)